src/components/ShowResultsSidebar.js

```diff
@@ -12,37 +13,42 @@ function ShowAttendees({ partitionedRespondents, time }) {
   // logged in, has admin rights, and whether the respondent has responded
   function renderRespondent(responder, respondent, attending) {
     const displayName = respondent.user ? respondent.user.name : respondent.anonymousName;
+
     return (
-      <div
-        className={classnames(
-          styles.respondents,
-          attending ? styles.borderAccept : styles.borderReject,
-        )}
-        key={responder}
-      >
-        {displayName}
-        <MaterialIcon icon="visibility" className={styles.icon} onClick={() => {}} />
-      </div>
+      <ListItem>
+        <ListItemGraphic graphic={<MaterialIcon icon='account_circle'/>} />
```

Comment:
I don't think we're supporting avatars yet right? We do need to add the key participant toggle though